src/com/owncloud/android/ui/activity/UploadFilesActivity.java

```diff
@@ -248,8 +248,14 @@ private String getCapturedImageName(){
     protected void onActivityResult(int requestCode,int resultCode,Intent capturedData){
         if(requestCode == REQUEST_IMAGE_CAPTURE && resultCode == RESULT_OK){
             new CheckAvailableSpaceTask(getCapturedImageFile().getAbsolutePath()).execute();
-        }
-        else if(resultCode == RESULT_CANCELED){
+        } else if(requestCode == REQUEST_IMAGE_CAPTURE && resultCode == RESULT_CANCELED){
+            if(image != null){
+                image.delete();
+                Log_OC.d(TAG,getResources().getString(R.string.file_deleted));
```

Comment:
We do not need to create string resources (that will be automatically translated) to log something, you can include the string itself here.